department-of-veterans-affairs/caseflow
# Pamatya/appeals 15446

Resolves [APPEALS-15446](https://vajira.max.gov/browse/APPEALS-15446)

# Description
1. In Disposition Component, passed the expected format for decisionDate. 
2. Added a feature test `decision_review_spec.rb` 
3. Added a jest test to test `formatDateStrUtc` method
4. Added a StoryBook For in-process and completed TaskPage Component.

## Acceptance Criteria
- [x] Decision dates appear in the decision date input field whenever visiting the task page for a completed decision review task.
- [x] Test coverage is added to protect against regressions
- [x] A storybook file for the TaskPage component is created that shows the Disposition child component.
## Testing Plan
1.  goto decision_reviews/vha
2.  Create  High level or supplemental Claim using intake new form button.
3.  search for veteran by id 
4.  select **Veterans health administrator** as benefit type, select other options and then click on continue. 
5.  on Add/Remove issue page, click on Add Issue button and  fill up necessary fills in the Modal. 
6. Click Next and then click **Establish Supplemental/HighLevel claim** button .
7. You should be redirected to decision_reviews/vha page with a new claim in in progress tab. 
8. Open the newly created claim from In Progress Tab.
9. Select Option in Disposition Dropdown, fill in Decision Description text area
10. Select Date in Date selector "remember the date", click on complete button. 
11. You Should be redirected to completed tab.
12. Click on the latest claim that you just completed. and scroll down to the button
13. You should see the date you selected in 10 to be present there as a read only option. 


 

# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
![image](https://user-images.githubusercontent.com/122557351/235724579-aa40d7f3-6e10-4883-ab67-75aefded1343.png)|![image](https://user-images.githubusercontent.com/122557351/235724937-453db3c3-ebdc-4399-83f7-052b12fb81bb.png)


## Storybook Story
*For Frontend (Presentationa) Components*



## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
